docs/source/miscellaneous/doc/adr/0016-use-tickets-encoding-for-transactions.md

```diff
@@ -499,7 +499,131 @@ the worker pool has a finite size and even with the full batching algorithm in t
 one requesting thread, and although CellLoader2 also spins up many more requesting threads on the Cassandra side,
 the Cassandra cluster was unable to actually have these therads all do work concurrently.
 
-### Live Migrations and the Coordination Service
+### Determining Which Transaction Service To Use
+
+Effectively, we want to define  an internal schema version for AtlasDB, and implement a mechanism for performing schema
```

Comment:
more of a question than a comment, what happens when you have a transaction that occurs over a schema version switch? i.e. start timestamp in the transactions 1 range, but commit timestamp in the tx2 range? I imagine we do things just in terms of the start timestamp?

Reply:
Yeah, that's correct. I will explicitly address this case in the doc.

- [ ] clarify starting before and finishing after a change-point case